src/winforms/toga_winforms/widgets/tree.py

```diff
@@ -0,0 +1,58 @@
+from toga_winforms.libs import WinForms
+from travertino.size import at_least
+
+from .base import Widget
+
+
+class Tree(Widget):
+    def create(self):
+        self.native = WinForms.TreeView()
+        self.headings = self.interface.headings
+
+    def change_source(self, source):
+
+        self.native.BeginUpdate()
+
+        def create_children(node):
+            if node._children is None or len(node._children) == 0:
+                # FIXME Works if there is only one accessor. Not sure what the behaviour
+                # should be with many accessors
+                return WinForms.TreeNode(getattr(node, self.interface._accessors[0]))
```

Comment:
On macOS and GTK, a Tree widget can still have multiple columns. So, you use the tree's +/- button to expand the tree's levels in the first column, but there can be additional columns showing other details. For example, the "Tree Source" example project shows multiple columns of detail for a movie, with the tree allowing you to sort by decade:
<img width="727" alt="screen shot 2018-03-06 at 2 18 28 pm" src="https://user-images.githubusercontent.com/37345/37016885-5212142e-2149-11e8-999a-c26a33963ff3.png">

The accessors then allow you to specify which attribute will be column 0, which will be column 1, and so on. Does that make sense?

Reply:
Thanks, this is what I needed. Will be working on it